crates/ethjson/src/transaction/transaction.rs

```diff
@@ -45,10 +47,37 @@ pub struct Transaction {
     pub v: Uint,
 }
 
+impl From<Transaction> for UnverifiedTransaction {
```

Comment:
I think this is removed from code because it is unused. Do you have usage for it?

Reply:
Afaik it’s used in tests. It’s also part of removing the dependency of ethcore on ethjson. I’m having lunch now and I’ll double check it ASAP when I get home.

Reply:
It was modified by dusan and removed when test started supporting AccessList, it is deadcode now.

Reply:
Sure, I'll remove it in a follow-up PR.